src/lib/Form/Type/Policy/PolicyChoiceType.php

```diff
@@ -18,6 +17,12 @@
 
 class PolicyChoiceType extends AbstractType
 {
+    const MESSAGE_DOMAIN = 'forms';
+    const MESSAGE_ID_PREFIX = 'role.policy.';
+    const ALL_MODULES = 'all_modules';
+    const ALL_FUNCTIONS = 'all_functions';
+    const ALL_MODULES_ALL_FUNCTIONS = 'all_modules_all_functions';
```

Comment:
Visibility should be declared explicitly, especially since those are private constants from what I see?

```suggestion
    private const MESSAGE_DOMAIN = 'forms';
    private const MESSAGE_ID_PREFIX = 'role.policy.';
    private const ALL_MODULES = 'all_modules';
    private const ALL_FUNCTIONS = 'all_functions';
    private const ALL_MODULES_ALL_FUNCTIONS = 'all_modules_all_functions';
```